<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qth_file</class>
 <widget class="QDialog" name="qth_file">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qth file</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="2">
        <widget class="QCheckBox" name="DMS_format_latitude">
         <property name="text">
          <string>DMS format</string>
         </property>
        </widget>
       </item>
       <item row="8" column="3">
        <widget class="QComboBox" name="longitude_choose">
         <item>
          <property name="text">
           <string>Eastern</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>West</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QCheckBox" name="DD_format_longitude">
         <property name="text">
          <string>DD format</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QLineEdit" name="longitude_qth"/>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="latitude_qth"/>
       </item>
       <item row="3" column="2">
        <widget class="QCheckBox" name="DD_format_latitude">
         <property name="text">
          <string>DD format</string>
         </property>
        </widget>
       </item>
       <item row="11" column="3">
        <widget class="QComboBox" name="metric">
         <item>
          <property name="text">
           <string>Feet</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Meters</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="latitude_label">
         <property name="text">
          <string>site's latitude</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="fila_label">
         <property name="text">
          <string>File name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QLineEdit" name="site_name"/>
       </item>
       <item row="10" column="2">
        <widget class="QCheckBox" name="DMS_format_longitude">
         <property name="text">
          <string>DMS format</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="name_label">
         <property name="text">
          <string>site's name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QComboBox" name="latitude_choose">
         <item>
          <property name="text">
           <string>Northern </string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>South</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="longitude_label">
         <property name="text">
          <string>site's longitude</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="height_label">
         <property name="text">
          <string>site's antenna height</string>
         </property>
        </widget>
       </item>
       <item row="11" column="2">
        <widget class="QLineEdit" name="height_qth"/>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="file_name">
         <property name="styleSheet">
          <string notr="true">combobox-popup: 0;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="Refresh">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="save_file_qth">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="save_as_file_qth">
         <property name="text">
          <string>Save as</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delete_file_qth">
         <property name="text">
          <string>Delete </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
